src/utils/patientIdentifierProcessor.ts

```diff
@@ -0,0 +1,21 @@
+import { PatientIdentifier } from "../api/types";
+import {editPatientIdentifier, savePatientIdentifier} from "../api/api";
+
+export const saveIdentifier = (patient: fhir.Patient, patientIdentifier:PatientIdentifier) =>{
```

Comment:
I would expect a "processor" to do more abstractions and processing than simply hosting helper functions.